include/librealsense2/h/rs_option.h

```diff
@@ -92,6 +92,7 @@ extern "C" {
         RS2_OPTION_INVALIDATION_BYPASS, /**< Enable\disable pixel invalidation */
         RS2_OPTION_AMBIENT_LIGHT, /**< Change the depth ambient light see rs2_ambient_light for values */
         RS2_OPTION_SENSOR_MODE, /**< The resolution mode: see rs2_sensor_mode for values */
+        RS2_OPTION_HARDWARE_SYNC_ENABLED, /**< If enabled, interrupts are handled according to GPIO10 level; shuts off the fall detect mechanism */
```

Comment:
The option RS2_OPTION_INTER_CAM_SYNC_MODE fits here, but requires a different implementation